changehc/delphi_changehc/run.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the function.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m delphi_changehc`.
+"""
+
+# standard packages
+import logging
+from datetime import datetime, timedelta
+from pathlib import Path
+
+#  third party
+from delphi_utils import read_params
+
+# first party
+from .update_sensor import CHCSensorUpdator
+
+
+def run_module():
+    """Run the delphi_changehc module.
+    """
+
+    params = read_params()
+
+    logging.basicConfig(level=logging.DEBUG)
+
+    ## get end date from input file
+    # the filenames are expected to be in the format:
+    # Denominator: "YYYYMMDD_All_Outpatients_By_County.dat.gz"
+    # Numerator: "YYYYMMDD_Covid_Outpatients_By_County.dat.gz"
+    
+    denom_suffix = params["input_denom_file"].split("/")[-1].split(".")[0][9:]
+    assert denom_suffix == "All_Outpatients_By_County"
```

Comment:
might be simpler to regex here but this works. can we add tests for this file?

Reply:
Tests added

Reply:
> Please drop the following files:
> 
> * `static/02_20_uszips.csv`
> * `tests/static/02_20_uszips.csv`
> * `params.json` (sorry about that; i completely spaced that both the code reviewer and the code writer were working in a relatively unfamiliar codebase)

Thanks Katie. There's a tests/params.json I think is ok to be commited (and needs to be set specifically for tests to pass)

Reply:
Realized i quoted the wrong comment here